src/components/v5/common/ActionSidebar/ActionSidebar.tsx

```diff
@@ -48,16 +50,17 @@ const ActionSidebarFormContent: FC<ActionSidebarFormContentProps> = ({
           stateClassNames={{
             [FIELD_STATE.Error]: 'placeholder:text-red-400',
           }}
-          className={clsx(
-            `
-              heading-3
-              md:hover:text-blue-400 md:hover:placeholder:text-blue-400 text-gray-900
-              transition-colors mb-7
-            `,
-          )}
+          className={`
+            heading-3
+            md:hover:text-blue-400 md:hover:placeholder:text-blue-400 text-gray-900
+            transition-colors
+          `}
           mode="secondary"
           message={false}
         />
+        <p className="text-gray-600 font-medium mt-2 mb-7">
```

Comment:
I think it will be better to move `mt-2` to Input above to keep margins on one way